pkg/osvscanner/osvscanner.go

```diff
@@ -341,7 +344,7 @@ func scanImage(r reporter.Reporter, path string) ([]scannedPackage, error) {
 
 // scanLockfile will load, identify, and parse the lockfile path passed in, and add the dependencies specified
 // within to `query`
-func scanLockfile(r reporter.Reporter, path string, parseAs string) ([]scannedPackage, error) {
+func scanLockfile(r reporter.Reporter, path string, parseAs string, allowResolver bool) ([]scannedPackage, error) {
```

Comment:
as above, I think this should be more "offline" themed as that is the important constraint - the current name reflects an implementation; people don't care about if resolvers are allowed, they care if any online actions will be taken